doc/tut2.rst

```diff
@@ -534,6 +534,19 @@ iterator or type. As the example shows, generics work with overloading: the
 best match of ``add`` is used. The built-in ``add`` procedure for sequences
 is not hidden and is used in the ``preorder`` iterator.
 
+There is a special [:T] syntax when using generics with universal function call syntax:
```

Comment:
It's called the "method call syntax", there is nothing "Universal" about special casing the first argument that a function receives.

Reply:
Noted. Thank you for the comment. I'm very new to Nim and wasn't exactly sure what to write there.